dags/challenge-submission-dag.py

```diff
@@ -0,0 +1,91 @@
+import asyncio
+
+from datetime import datetime
+
+from airflow.decorators import dag, task
+from airflow.models import Param
+
+from orca.services.nextflowtower import NextflowTowerHook
+from orca.services.nextflowtower.models import LaunchInfo
```

Comment:
Because `LaunchInfo` is a user-facing model, we can import it into the `nextflowtower` module (in the `__init__.py` file) so it doesn't require its own import line. Just an idea for the next time that we update py-orca.